tools/mason/MasonSearch.chpl

```diff
@@ -0,0 +1,78 @@
+
+use MasonHelp;
+use MasonUpdate;
+
+use FileSystem;
+use Regexp;
+
+//
+// TODO:
+// - order results by some kind of 'best match' metric
+// - allow regex searches
+// - allow for exclusion of a pattern
+//
+
+proc masonSearch(origArgs : [] string) {
+  var args : [1..origArgs.size] string = origArgs;
+
+  const printHelp = || reduce [a in args] (a == "-h" || a == "--help");
```

Comment:
Should `--no-update-registry` be a flag displayed in the help output?

Reply:
I don't think so, for now it's just for testing.